project-base/storefront/components/Blocks/UserConsent/UserConsentForm.tsx

```diff
@@ -6,7 +6,6 @@ import { onGtmConsentUpdateEventHandler } from 'gtm/helpers/eventHandlers';
 import { getGtmConsentInfo } from 'gtm/helpers/gtm';
```

Comment:
I know this is not an issue introduced by you (was actually probably initially written by me), but could I ask you to unify the naming in this file? We use `cookies` and `consent` and it might be confusing.

Reply:
I recommend create new task for this because it goes further and scope of this task is already far behind actual work. There is also URQL hook, article name, maybe other places. So better to check this as whole.